pkg/controller/zerocapacity/zero_controller.go

```diff
@@ -328,11 +328,14 @@ func (z *Controller) zeroPodSpec(name, namespace, configMap string, podSecurityC
 		return nil, err
 	}
 	dataVolName := name + "-data"
+	// Add Custom labels missing from zero capacity pods
```

Comment:
No need for the comment as it won't be missing with your changes :slightly_smiling_face: